homeassistant/components/homewizard/strings.json

```diff
@@ -27,5 +27,145 @@
       "unknown_error": "[%key:common::config_flow::error::unknown%]",
       "reauth_successful": "Enabling API was successful"
     }
+  },
+  "entity": {
+    "button": {
+      "status_light_brightness": {
+        "name": "Status light brightness"
+      }
+    },
+    "sensor": {
+      "dsmr_version": {
+        "name": "DSMR version"
+      },
+      "meter_model": {
+        "name": "Smart meter model"
+      },
+      "unique_meter_id": {
+        "name": "Smart meter identifier"
+      },
+      "wifi_ssid": {
+        "name": "Wi-Fi SSID"
+      },
+      "active_tariff": {
+        "name": "Active tariff"
+      },
+      "wifi_strength": {
+        "name": "Wi-Fi strength"
```

Comment:
Strength is in percentage, not RSSI (as commonly used). Can we make that clear during translation somehow?

Reply:
The unit of measurement should be set to percentage then, I'll take a look

Reply:
Yeah better. The `key` is matching the key in the API, where we do not include the unit (legacy). But for the translation_key it may be nice.

Reply:
the unit is already percentage, so it will automatically show percentage as unit.

Reply:
You are right Joost. The suggested translation string here, pretty much in line with others. LGTM 👍